### PR TITLE
Movie seat booking

### PR DESCRIPTION
**Pull request by natashafir**
[Demo](https://natashafir.github.io/cinema/)
[Code](https://github.com/natashafir/cinema)

### CODE REVIEW

**Comment by krylenger** [comment]
Indeed Great job!

Some things to improve: 
-when I navigate through seats using keyboard, I can't see my current position. (which seat is now in focus) 
-on mobile device the description section is not displayed

File: submissions/natashafir/html-movie-seat-booking/index.html [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
+</head>
+<body>
+<div class="wrapper">
+    <div class="hall">
+        <div class="screen"></div>
+        <div class="all-seats">
+            <div class="firsts-seats">
+
+                <input aria-label="row 1 seat 1" type="checkbox" id="seat-1-1" class="visually-hidden">
```
Comment (krylenger) [inline submissions/natashafir/html-movie-seat-booking/index.html]: add value attributes to your input tags
[checkbox value](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#value)

File: submissions/natashafir/html-movie-seat-booking/style.css [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+    color: #e3e6ea;
+    font-size: 12px;
+    text-align: center;
+    line-height: 2;
+}
+
+label.not-available {
+    background-color: #2d4059;
+    color: #43658b;
+}
```
Comment (krylenger) [inline submissions/natashafir/html-movie-seat-booking/style.css]: why do you need here label.className ? 
you can use just .className to not specify your selection

**Comment by krylenger** [comment]
one of the main aims for this task is to implement semantic HTML.
try to change some of your divs with semantic tags

File: submissions/natashafir/html-movie-seat-booking/index.html [hunk 3]
```diff
@@ -0,0 +1,325 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Booking tickets</title>
+    <link href="style.css" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
+</head>
+<body>
```
Comment (A-Ostrovnyy) [inline submissions/natashafir/html-movie-seat-booking/index.html:13]: Remove horizontal scroll on mobile

image

File: submissions/natashafir/html-movie-seat-booking/style.css (hunk 2 above)
Comment (A-Ostrovnyy) [reply submissions/natashafir/html-movie-seat-booking/style.css]: Avoid using too specific selectors.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
See my comments below!

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**